manifest.json

```diff
@@ -301,6 +301,13 @@
           "js": ["d3.min.js"]
         }
       },
+      "diff-match-patch": {
+        "version": "1.0.0",
```

Comment:
Optional: would it be better, for consistency, to download a file from https://github.com/google/diff-match-patch using the "commit hash" method?

Reply:
Oh hm, seems you did that last time. What went wrong when you did that?

Reply:
Sorry for the delay in response, the JS file in GitHub repository has the function through a variable which is not used by codemirror and i've used the same link to download the library which is used in the codemirror files with <script src.
Please take a look at the below picture
<img width="836" alt="Screenshot 2019-06-25 at 3 10 45 PM" src="https://user-images.githubusercontent.com/41017303/60101816-1f0dcd00-9765-11e9-83eb-d9ffb5983991.png">